src/sqlcode.h

```diff
@@ -1,12 +1,10 @@
 /******************************************************************************
  *
- * 
- *
- * Copyright (C) 1997-2014 by Dimitri van Heesch.
+ * Copyright (C) 1997-2020 by Dimitri van Heesch.
```

Comment:
What is the relevance of the changes in this file in respect to the hyphenation in tables?